bionic/flow.py

```diff
@@ -460,8 +469,10 @@ def assign(
             protocol.
         doc: String, optional
             Description of the new entity.
-        persist: Boolean, optional
+        persist: Boolean, optional (default: True)
             Whether this entity's values should be cached persistently.
+            Set it to False if an internal Bionic entity depends on it.
```

Comment:
I would say "The only reason to set this to False is if an internal Bionic entity depends on it; in this case, persistence is impossible because Bionic's cache won't be constructed by the time this entity is calculated. The downside of setting this to False is that it won't be possible to retrieve a serialized file for this entity using the ``mode`` argument to ``Flow.get``."

This avoids the ambiguity of "Otherwise" below, and also explains why you would ever *not* set it to False.